src/docker-compose.yml

```
@@ -51,8 +51,29 @@ services:
       interval: 15s
       timeout: 2s
       retries: 5
-    # devices:
-    #   - "/dev/i2c-1:/dev/i2c-1"  
+    devices:
+      - "/dev/i2c-1:/dev/i2c-1"  
+    environment:
+      - ONRASPBERRY=${ONRASPBERRY}
+
+  raspberrydevconn:
+    build: 
+      context: ./
+      dockerfile: ./services/raspberrydevconn/Dockerfile
+    ports:
+      - "8088:8088"
+    depends_on:
+      catalog:
+          condition: service_started
+      deviceconfig:
+          condition: service_started
+    healthcheck:
+      test: ["CMD-SHELL", "curl -f -Li catalog:8080/catalog/services/raspberrydevconn | grep online | grep -c true"]
+      interval: 15s
+      timeout: 2s
+      retries: 5
+    devices:
+      - "/dev/i2c-1:/dev/i2c-1"  
```

```suggestion
    #devices:
    #  - "/dev/i2c-1:/dev/i2c-1"  
```

I suggest to comment out this when you commit so the GH Action doesn't fail

---

Yes, you are right. i noticed only after the action failed!